testing/simulated/pectra_fork_test.go

```diff
@@ -186,20 +190,25 @@ func (s *PectraForkSuite) TestTimestampFork_ELAndCLInSync_IsSuccessful() {
 		"Processing block with fork version service=blockchain\u001B[0m block=2\u001B[0m fork=0x04010000\u001B[0m",
 		"Processing block with fork version service=blockchain\u001B[0m block=3\u001B[0m fork=0x04010000\u001B[0m",
 		"Processing block with fork version service=blockchain\u001B[0m block=4\u001B[0m fork=0x04010000\u001B[0m",
-		"Processing block with fork version service=blockchain\u001B[0m block=5\u001B[0m fork=0x05000000\u001B[0m",
-		"Processing block with fork version service=blockchain\u001B[0m block=6\u001B[0m fork=0x05000000\u001B[0m",
-		"Processing block with fork version service=blockchain\u001B[0m block=7\u001B[0m fork=0x05000000\u001B[0m",
+		"Processing block with fork version service=blockchain\u001B[0m block=5\u001B[0m fork=0x04010000\u001B[0m",
+		"Processing block with fork version service=blockchain\u001B[0m block=6\u001B[0m fork=0x04010000\u001B[0m",
+		"Processing block with fork version service=blockchain\u001B[0m block=7\u001B[0m fork=0x04010000\u001B[0m",
+		"Processing block with fork version service=blockchain\u001B[0m block=8\u001B[0m fork=0x04010000\u001B[0m",
+		"Processing block with fork version service=blockchain\u001B[0m block=9\u001B[0m fork=0x05000000\u001B[0m",
+		"Processing block with fork version service=blockchain\u001B[0m block=10\u001B[0m fork=0x05000000\u001B[0m",
+		"Processing block with fork version service=blockchain\u001B[0m block=11\u001B[0m fork=0x05000000\u001B[0m",
 	}
 
-	startHeight := int64(1)
-	iterations := int64(len(expectedMessages))
-	expectedMessagesIdx := 0
-	submitTxNonce := uint64(0)
+	var (
+		startHeight         = int64(1)
+		iterations          = int64(len(expectedMessages))
+		expectedMessagesIdx = 0
+		submitTxNonce       = uint64(0)
+		consensusTime       = time.Unix(startHeight*2, 0)
```

Comment:
Minor rearrangement due to optimistic block building. Set time for next block at the end of the for loop, instead of setting time for current block at the start of the loop